src/main/java/Ui.java

```diff
@@ -0,0 +1,23 @@
+import task.TaskList;
+
+public class Ui {
+    public static void print(String... strings) {
+        System.out.println(Messages.LINE.MESSAGE);
+        for (String s : strings) {
```

Comment:
Perhaps it might be clearer to use "string" instead of "s", as in "for (String string : strings)".